cronjobs/create_pipeline_execution.py

```diff
@@ -0,0 +1,231 @@
+        "populate_zeder_journal" : {"parent_to_child_linking"},
+        "add_additional_open_access" : {"populate_zeder_journal"},
+        "extract_normdata_translations" : {"add_beacon_to_authority_data"},
+        "add_author_synomyns_from_authority" : {"add_additional_open_access"},
+        "add_aco_fields" : {"add_author_synomyns_from_authority"},
+        "add_isbn_issn" : {"add_aco_fields"},
+        "extract_keywords_from_titles" : {"add_isbn_issn"},
+        "flag_electronic_and_open_access" : {"extract_keywords_from_titles"},
+        "augment_bible_references" : {"flag_electronic_and_open_access"},
+        "augment_canon_law_references" : {"augment_bible_references"},
+        "augment_time_aspect_references" : {"augment_canon_law_references"},
+        "update_ixtheo_notations" : {"augment_time_aspect_references"},
+        "replace_689a_689q" : {"update_ixtheo_notations"},
+        "map_ddc_to_ixtheo_notations" : {"replace_689a_689q"},
+        "add_keyword_synonyms_from_authority" : {"add_beacon_to_authority_data", "map_ddc_to_ixtheo_notations"},
+        "fill_missing_773a" : {"add_keyword_synonyms_from_authority"},
+        "tag_further_potential_relbib_entries" : {"fill_missing_773a"},
+        "integrate_sort_year_for_serials" : {"tag_further_potential_relbib_entries"},
+        "integrate_refterms" : {"integrate_sort_year_for_serials"},
+        "tag_tue_records_with_ita_field" : {"integrate_refterms"},
+        "add_entries_for_subscription_bundles" : {"tag_tue_records_with_ita_field"},
+        "add_tags_for_subsystems" : {"add_entries_for_subscription_bundles"},
+        "appending_literary_remains" : {"add_beacon_to_authority_data", "add_tags_for_subsystems"},
+        "tag_pda_candidates" : {"add_tags_for_subsystems"},
+        "patch_transitive_records" : {"tag_pda_candidates"},
+        "cross_link_type_tagging" : {"patch_transitive_records"},
+        "tag_inferior_records" : {"cross_link_type_tagging"},
+        "check_record_integrity_end" : {"extract_translations", "tag_inferior_records"},
+        "cleanup" : {"check_record_integrity_end", "extract_translations"}
+       }
+
+def get_phase_by_key(key):
+    for phase in phases:
+        if phase.key == key:
+            return phase
+    return None
+
+def process_chunk(lst):
+    global phase_counter
+    if len(lst) > 0:
+        for i, elem in enumerate(lst):
+            phase_counter += 1
+            if len(lst) > 1 and i != len(lst) - 1:
+                print("making fifo - ", elem.title_norm, " phase counter: ", phase_counter)
+            print("processing ", elem.key, " -> ", elem.title_norm, " phase counter: ", phase_counter)
+        print("---")
+
+def split_and_write_pipeline_steps(lst, n):
+    open_fifo_title = []
+    open_fifo_norm =[] 
+
+    for elemkey in lst:
+        elem = get_phase_by_key(elemkey)
+        phase_mode = elem.mode
+        phase_title_norm = elem.title_norm
+
+        if phase_title_norm.startswith("x"):
```

Comment:
When comparing phase_title_norm, you should use == instead of startswith. startswith suggests that the string may have multiple characters, which is allowed for phase_mode, but not for phase_title_norm.